src/gui/DatabaseSettingsWidget.ui

```diff
@@ -6,225 +6,220 @@
    <rect>
```

Comment:
You broke the whole layout which we carefully fixed in KeePassXC. Please consider a clean re-implementation based on our UI file instead of trying to auto-merge your changes.